src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -94,10 +104,27 @@ private static void OnProcessExit(object sender, EventArgs e)
             }
         }
 
+        private void LogEventRaised(NativeInterop.Core sender, LogEventArgs args)
+        {
+            using LogEventWrapper logEventWrapper = new LogEventWrapper(this);
```

Comment:
This should be removed since LogEventWrapper constructor adds LogEventRaised to the static event . So each time LogEventRaised is raised, we're adding one more handler.

Reply:
Good catch. That was copy/paste for each method added here too.